lib/src/firebase/firebaseClient.dart

```diff
@@ -33,4 +33,83 @@ class FirebaseClient {
   Future _onAuthChanged(fb.User user) async {
     print("Auth Changed :: $user");
   }
+
+  // getMembers (with role or all?)
+  getMembers(){
+    return _refs.userRef().get();
+  }
+  // getMeals (by date/date range?)
+  getMeals(){
+    return _refs.userRef().get();
+  }
+  // getClasses (by day, range?)
+  getClasses(){
+    return _refs.userRef().get();
+  }
+
+  //create new user/class/meal
+  createUser(name, email, phone, password, role, classes){
```

Comment:
Dart functions need types in arguments, so `createUser(String name, int phone,...)` yadda yadda